cli/config/config.go

```diff
@@ -254,22 +258,36 @@ func (c *CommonOpts) GetServiceCredential() *upstream.ServiceAccountCredentials
 
 	return &upstream.ServiceAccountCredentials{
 		Mrn:         c.ServiceAccountMrn,
-		ParentMrn:   c.GetParentMrn(),
+		ParentMrn:   c.GetScopeMrn(),
+		ScopeMrn:    c.GetScopeMrn(),
 		PrivateKey:  c.PrivateKey,
 		Certificate: c.Certificate,
 		ApiEndpoint: c.APIEndpoint,
 	}
 }
 
-func (c *CommonOpts) GetParentMrn() string {
-	parent := c.ParentMrn
+// GetScopeMrn returns the scope mrn that is used for the service account.
+// This is either the organization mrn or the space mrn.
+func (c *CommonOpts) GetScopeMrn() string {
+	scopeMrn := c.ScopeMrn
 
 	// fallback to old space_mrn config
-	if parent == "" {
-		parent = c.SpaceMrn
+	if scopeMrn == "" {
+		scopeMrn = c.SpaceMrn
+	}
+
+	if scopeMrn == "" {
+		scopeMrn = c.ParentMrn
 	}
 
-	return parent
+	return scopeMrn
+}
+
+// GetParentMrn returns the scope mrn that is used for the service account.
+// This is either the organization mrn or the space mrn.
+// Deprecated: Use GetScopeMrn instead
+func (c *CommonOpts) GetParentMrn() string {
```

Comment:
can we just not remove this function? i dont think this is used by an end user in some way so should be safe to delete it and migrate cnspec to `GetScopeMrn` (if cnspec makes use of that)

Reply:
Lets leave it deprecated for now.